util/helm/cmd.go

```diff
@@ -132,40 +220,125 @@ func writeToTmp(data []byte) (string, io.Closer, error) {
 	}), nil
 }
 
-func (c *Cmd) Fetch(repo, chartName, version, destination string, creds Creds) (string, error) {
-	args := []string{c.pullCommand, "--destination", destination}
+func (c *Cmd) Fetch(repoNamespace, repo, chartName, version, destination string, creds Creds) (string, error) {
```

Comment:
General thoughts about the new code in this method: There's lots of duplicate code in the execution trees for when repo is OCI vs when it's not, maybe it's generally a good idea to compact the duplicate code into its own method. Would make later maintenance a little easier.

Reply:
agree